src/engine.rs

```diff
@@ -168,9 +168,6 @@ impl Drop for Reedline {
         if self.bracket_paste_enabled {
             let _ = execute!(io::stdout(), DisableBracketedPaste);
         }
-        if self.use_kitty_protocol {
-            let _ = execute!(io::stdout(), event::PopKeyboardEnhancementFlags);
-        }
```

Comment:
To be defensive we could keep this in case the application hosting reedline panics.

Reply:
Oh that's good idea